src/js/providers/html5.js

```diff
@@ -482,9 +482,6 @@ define([
                 // Playback rate is broken on Android HLS
                 _this.supportsPlaybackRate = false;
             }
-            if (source.preload && source.preload !== _videotag.getAttribute('preload')) {
-                _setAttribute('preload', source.preload);
-            }
```

Comment:
Good point. Since we no longer want to trigger preload on `init`, does it make sense to set it to `none` here, and then update it when the preloading conditions are met?